tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -647,6 +648,41 @@ public async Task GetAuthorizationRequestUrlNoRedirectUriTestAsync()
             }
         }
 
+        [TestMethod]
+        public async Task GetAuthorizationRequestUrlWithPKCETestAsync()
```

Comment:
I think we need an E2E test that covers both `GetAuthorizationRequestUrl` and `AcquireTokenByAuthCode`

Reply:
I will add an integration test for this. We should also have a unit test for AcquireTokenByAuthorizationCode with pkce